unityparser/scanner.py

```diff
@@ -28,3 +29,31 @@ def get_token(self):
             # UNITY: keep track of previous tokens
             self.prev_tokens.append(self.tokens.pop(0))
             return self.prev_tokens[-1]
+
+    def fetch_anchor(self):
+        super().fetch_anchor()
+
+        # after fetching the anchor, we need to determine if we are at
+        # end of line, or if there are further tokens past the anchor.
+        # If at the end of line, we can continue as normal.  If there
+        # are more tokens (like the word 'stripped') then we need
+        # to grab that token and save it, then we can continue
+        if self.peek() in '\r\n\x85\u2028\u2029':
```

Comment:
I'd rather have a `scan_extra_anchor_data` method to do the actual parsing to keep consistency with the base library.
Also remove commented code, I guess it's stale code from other alternatives ?

Reply:
Resolved.  I moved fetch_anchor into the unityparser derived class and called the extra parsing from there.  Will be in next PR